cloudinit/sources/DataSourceOpenStack.py

```diff
@@ -29,7 +29,8 @@
 DMI_PRODUCT_COMPUTE = 'OpenStack Compute'
 VALID_DMI_PRODUCT_NAMES = [DMI_PRODUCT_NOVA, DMI_PRODUCT_COMPUTE]
 DMI_ASSET_TAG_OPENTELEKOM = 'OpenTelekomCloud'
-VALID_DMI_ASSET_TAGS = [DMI_ASSET_TAG_OPENTELEKOM]
+DMI_ASSET_TAG_SAPCCLOUD = 'SAP CCloud VM'
```

Comment:
Is there any documentation we could link to for this value? (I totally understand if not, but would be good to have if possible.)

Reply:
The "documentation" is here https://github.com/sapcc/helm-charts/blob/master/openstack/nova/values.yaml#L242

Reply:
Aha, I poked around in the nova project over there, didn't think to look in helm-charts. Even a pointer to that would be good (ideally using a permalink, so https://github.com/sapcc/helm-charts/blob/c72c8a40a68f718a372607c2b1984a6f91f74291/openstack/nova/values.yaml#L242).